homeassistant/components/thermopro/manifest.json

```diff
@@ -24,5 +32,5 @@
   "dependencies": ["bluetooth_adapters"],
   "documentation": "https://www.home-assistant.io/integrations/thermopro",
   "iot_class": "local_push",
-  "requirements": ["thermopro-ble==0.10.1"]
+  "requirements": ["thermopro-ble==0.11.0"]
```

Comment:
Please bump the dep in a separate PR

Reply:
Done - PR https://github.com/home-assistant/core/pull/137381 - these changes will be dropped as soon as that PR is merged.